apps/meteor/app/ui-utils/client/lib/MessageAction.ts

```diff
@@ -11,16 +11,16 @@ import { TranslationKey } from '@rocket.chat/ui-contexts';
 import { Messages, Rooms, Subscriptions } from '../../../models/client';
 import { roomCoordinator } from '../../../../client/lib/rooms/roomCoordinator';
 import { ToolboxContextValue } from '../../../../client/views/room/lib/Toolbox/ToolboxContext';
-
-const call = (method: string, ...args: any[]): Promise<any> =>
-	new Promise((resolve, reject) => {
-		Meteor.call(method, ...args, function (err: any, data: any) {
-			if (err) {
-				return reject(err);
-			}
-			resolve(data);
-		});
-	});
+import { APIClient } from '../../../utils/client';
+
+const getMessage = async (msgId: string): Promise<any> => {
+	try {
+		const { message } = await APIClient.get('/v1/chat.getMessage', { msgId });
+		return message;
+	} catch {
+		return null;
+	}
+};
```

Comment:
we were rejecting if there was an error, and where we are suppressing the error, is that correct?